pkg/kv/kvserver/rangefeed/catchup_scan.go

```diff
@@ -141,7 +141,11 @@ type outputEventFn func(e *kvpb.RangeFeedEvent) error
 // TODO(sumeer): ctx is not used for SeekGE and Next. Fix by adding a method
 // to SimpleMVCCIterator to replace the context.
 func (i *CatchUpIterator) CatchUpScan(
-	ctx context.Context, outputFn outputEventFn, withDiff bool, withFiltering bool,
+	ctx context.Context,
+	outputFn outputEventFn,
+	withDiff bool,
```

Comment:
note that I plan to follow up this PR with a patch that puts these with options into a options slice for these various call sites.